src/Internal/Metadata/Helpers.purs

```diff
@@ -36,3 +38,11 @@ lookupMetadata _ _ = Nothing
 errExpectedObject :: forall (a :: Type). Either JsonDecodeError a
 errExpectedObject =
   Left (TypeMismatch "Expected object")
+
+chunksOf :: forall a. Int -> Array a -> Array (Array a)
+chunksOf _ [] = []
+chunksOf n xs =
+  let
+    { after, before } = splitAt n xs
+  in
```

Comment:
This solution is not acceptable, because it assumes ASCII.

Please just replace String type with `Ctl.Internal.Metadata.Cip25.Cip25String` - all the machinery to properly split the string is already here.

Reply:
I went ahead and did this.

One issue that this created is that the `fromMetadataString` function expected that, if a string had been made into an array, that it would be an array of `ByteArray`s, when it's actually an array of `Cip25String`s. I changed the `fromMetadataString` function accordingly.